docs/modules/SDDs/pages/0022-managed-services-in-cluster.adoc

```diff
@@ -0,0 +1,158 @@
+
+[NOTE]
+.Summary
+====
+Besides the system services Project Syn also provides additional services to the user of the platform. This SDD describes how some kind of managed services on Kubernetes with Project Syn can be achieved. It's considered to be the initial version of this concept.
+====
+
+== Motivation
+
+While the current state of Project Syn focuses on delivering system services to Kubernetes clusters, it should also take application services into consideration. There needs to be a way to install, configure and maintain application services and make them available to the user of the platform.
+
+As application services are considered services like databases, caches, queues and other similar applications. The're usually used by the end user application running on a Kubernetes cluster, but could also be used by applications running outside of the cluster (f.e. in another cluster or even a VM).
+
+This doesn't include services offered by the cloud.
+
+=== Goals
+
+* Offer application services to applications running in or outside of the cluster
+* Maintenance processes to maintain these application services, making them a managed service
+* Definition of how the end user application instantiates and consumes an application service
+* Integration into backup and restore processes
+* Automatic monitoring and alerting of service instances
+
+=== Non-Goals
+
+* Services offered by the cloud aren't part of this SDD
+* All plumbing and system services are discussed in other SDDs
+
+== Design Proposal
+
+It's assumed that all application services are managed by a specialized Kubernetes Operator, which brings the domain knowledge about the specific service. This Kubernetes Operator will be installed and configured by a Commodore Component. The Commodore Component will be responsible for:
+
+* Installing the Kubernetes Operator
+* Pre-configuring the Kubernetes Operator with best practices, if applicable
+* Deploy additional objects like monitoring definition, backup definition or any other object needed to make the application service a managed service
+
+When the Kubernetes Operator is installed and operational, the user of the platform requests an instance of the application service by creating an Operator specific Kubernetes object. The job of the Kubernetes Operator is to instantiate the service and provide the information to the user for actually connecting to the service.
+
+This SDD doesn't define how this service instance object is handled and managed. It could come together with the application deployment objects of the customer but also being stored in a separate GitOps repository just for these objects.
+
+A Kubernets Operator could also be responsible to manage different kinds of application services, it'sn't strictly necessary that each Operator is only responsible for one service.
+
+Each provided service will be documented in the Commodore Component which installs the Kubernetes Operator for the service.
```

Comment:
Not sure if that's the right place to document a service?
I think such a documentation would outgrow a component in terms of details. It's also very possible that multiple components are required for a single service.
```suggestion
Each provided service will be documented in a dedicated documentation repo for provided services.
```